pollux/core/src/main/scala/org/hyperledger/identus/pollux/core/model/error/CredentialServiceError.scala

```diff
@@ -120,7 +120,13 @@ object CredentialServiceError {
         StatusCode.NotFound,
         s"A key with the given purpose was not found in the DID: did=${did.toString}, purpose=${verificationRelationship.name}"
       )
-
+  final case class MultipleKeysWithSamePurposeFoundInDID(
```

Comment:
@mineme0110, is it possible to keep two keys (secp256 and de25519) with the same purpose?

Reply:
https://github.com/hyperledger/identus-cloud-agent/issues/1422

Reply:
Yes it is possible to have multiple  keys with same purpose as per the spec